scheduler/cloudformation/cloudformation.go

```diff
@@ -235,34 +246,70 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 		return &templateValidationError{templateURL: url, err: err, templateBody: buf}
 	}
 
+	var stackParameters []*cloudformation.Parameter
+	var create bool
+	resp, err := s.cloudformation.DescribeStacks(&cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	})
+	if err, ok := err.(awserr.Error); ok && err.Message() == fmt.Sprintf("Stack with id %s does not exist", stackName) {
+		create = true
+	} else if err == nil {
+		create = false
+		stackParameters = resp.Stacks[0].Parameters
```

Comment:
This is probably a dangerous place to do this. If this is an update, it might get queued behind an in progress update, at which point these parameters may have been invalidated from that update. The parameters returned from describing a stack should only really be considered valid while the process holds the [stackOperation lock](https://github.com/mhahn/empire/blob/eb40679ae5416bc9d2c4e636f3485b185c44d0aa/scheduler/cloudformation/cloudformation.go#L393-L397).

When the stack update eventually gets executed, [then it will describe the stack and merge in the existing stack parameters there](https://github.com/mhahn/empire/blob/eb40679ae5416bc9d2c4e636f3485b185c44d0aa/scheduler/cloudformation/cloudformation.go#L455-L486).